Editor/Window/ManagedEC2Page.cs

```diff
@@ -39,7 +39,7 @@ public ManagedEC2Page(VisualElement container, StateManager stateManager)
                             $"{Application.productName}-{_deploymentSettings.ScenarioName.Replace(" ", "_")}-Build",
                 GameServerFile = _deploymentSettings.BuildFilePath,
                 GameServerFolder = _deploymentSettings.BuildFolderPath,
-                OperatingSystem = FleetParametersInput.GetOperatingSystem(_deploymentSettings.BuildOperatingSystem) ??
+                OperatingSystem = OperatingSystem.FindValue(_deploymentSettings.BuildOperatingSystem) ??
```

Comment:
Can we clean up the `OSMappings` enum if that is no longer needed/used?

Reply:
`OSMappings` is still used within `FleetParametersInput` to set up the dropdown values https://github.com/aws/amazon-gamelift-plugin-unity/blob/9b2b516e608e13fc9992d543236c8e31e4020806/Editor/Window/FleetParametersInput.cs#L55-L56